### ISSUE TITLE
(in)validation de quêtes via rechargement

### ISSUE THREAD

**Issue by xelinchaud** [JulienNigon/SpellcasterUniversity-Bug-Report#1003]
Bonjour,
J'ai rechargé une partie de la campagne à l'écran de sélection finale d'une carte avec l'arrivée du chaos(juste avant le choix d'avenir des élèves) et la quête de réputation orque a était invalidée, me montrant une étoile partiellement jaune plutôt que pleinement et ne me donnant pas le grimoire niveau 3 alors que j'avais les atteint les 90 de réputations au cours de cette partie. J'ai rencontré un bug similaire pour une quête d'archimage qui a était validée sans archimage suite au même type de chargement.
P.S: J'ai tendance a fermer le jeu avec alt-F4 ce qui est probablement la cause du bug.

**Comment by Neelan22** [comment]
Non, ce n'est pas à cause de alt F4, la quête de l'orc n'est pas bien sauvegardée, il faut qu'on corrige ça. Actuellement, il faut pouvoir la revalider après avoir repris une sauvegarde.

**Comment by scpsc** [comment]
Je viens d'avoir le problème avec la quête orc : avant de quitter la partie, je l'avais validée, puis je me souviens avoir perdu de la réputation en les envoyant paître, ce qui n'avait pas changé la couleur de l'étoile. A mon retour dans la partie, je suis à 80/90 et l'étoile n'est plus validée.

Après l'avoir validée à nouveau, je n'ai pas eu la visite de l'orc pour m'annoncer la bonne nouvelle. Je suppose que c'était donc peut-être pas la peine de remonter les 10 points manquants : le jeu avait peut-être gardé en mémoire la validation et ce ne serait qu'un problème d'affichage ?

**Comment by Neelan22** [comment]
Non, il fallait bien la revalider. Là ça ressemble à un bug sur un bug.

**Comment by Hykus1605** [comment]
J'ai eu le même problème.

J'ai validé la quête orc puis ma réputation à baisser. j'ai sauvegarder et quitter et a mon retour la quête n'est plus validé (je n'ai pas pu la revalider car c'était la fin de la partie).

Elle n'est effectivement pas gardé en mémoire car je n'ai eu qu'un grimoire de niveau 2 (mes deux autres quêtes étaient validées)
